docs/usage/configuration-options.md

```diff
@@ -2885,18 +2885,25 @@ It is also recommended to avoid `rebaseWhen=never` as it can result in conflicte
 
 Avoid setting `rebaseWhen=never` and then also setting `prCreation=not-pending` as this can prevent creation of PRs.
 
-## recreateClosed
+## recreateWhen
 
-By default, Renovate will detect if it has proposed an update to a project before and not propose the same one again.
-For example the Webpack 3.x case described above.
-This field lets you customize this behavior down to a per-package level.
-For example we override it to `true` in the following cases where branch names and PR titles need to be reused:
+By default, Renovate detects if it proposed an update to a project before, and will not propose the same update again.
```

Comment:
```suggestion
This feature used to be called `recreateClosed`.

By default, Renovate detects if it proposed an update to a project before, and will not propose the same update again.
```

Maybe we should add a note here, so that users searching for the old `recreateClosed` term can find the replacement?